minet/cli/twitter/tweets.py

```diff
@@ -0,0 +1,77 @@
+# =============================================================================
+# Minet Twitter Tweets CLI Action
+# =============================================================================
+#
+# Logic of the `tw tweets` action.
+#
+import casanova
+from twitwi import (
+    normalize_tweet,
+    format_tweet_as_csv_row
+)
+from twitwi.constants import TWEET_FIELDS
+from twitter import TwitterHTTPError
+
```

Comment:
This empty line can be removed. We only separate the lib imports from the inner ones.